src/injector/src/main.rs

```diff
@@ -86,90 +95,82 @@ fn unpack(sha_sum: &String) {
         .expect("Unable to unarchive the resulting tarball");
 }
 
+
+
 /// Starts a static docker compliant registry server that only serves the single image from the CWD
 ///
 /// (which is a OCI image layout):
 ///
 /// index.json - the image index
 /// blobs/sha256/<sha256sum> - the image layers
 /// oci-layout - the OCI image layout
-fn start_seed_registry() {
-    let root = PathBuf::from("/zarf-seed");
-    println!("Starting seed registry at {} on port 5000", root.display());
-    rouille::start_server("0.0.0.0:5000", move |request| {
-        rouille::log(request, io::stdout(), || {
-            router!(request,
-                (GET) (/v2/) => {
-                    // returns empty json w/ Docker-Distribution-Api-Version header set
-                    Response::text("{}")
-                        .with_unique_header("Content-Type", "application/json; charset=utf-8")
-                        .with_additional_header("Docker-Distribution-Api-Version", "registry/2.0")
-                        .with_additional_header("X-Content-Type-Options", "nosniff")
-                },
-
-                _ => {
-                    handle_request(&root, &request)
-                }
-            )
-        })
-    });
+fn start_seed_registry() -> Router{
+    // The name and reference parameter identify the image
+    // The reference may include a tag or digest.
+    Router::new()
+    .route("/v2/*path", get(handler))
+    .route("/v2/", get(|| async { 
+    Response::builder()
+        .status(StatusCode::OK)
+        .header("Content-Type", "application/json; charset=utf-8")
+        .header("Docker-Distribution-Api-Version", "registry/2.0")
+        .header("X-Content-Type-Options", "nosniff")
+        .body(Body::empty())
+        .unwrap()
+    }))
+    .route("/v2", get(|| async { 
```

Comment:
I think axum will route these to the same place so we can probably drop the second?

https://github.com/tokio-rs/axum/issues/1118